platform/lib/templates/ImportBlogFilter.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const POST_COUNT = 6;
+const BLOG_PATH = `https://blog.amp.dev/wp-json/wp/v2/posts?per_page=${POST_COUNT}&_embed`;
+const DEFAULT_IMG = 'AMP_Blog_Square.jpg';
+const fetch = require('node-fetch');
+const moment = require('moment');
```

Comment:
As we're not using `moment` anywhere else and Wordpress delivers a well-formed date string it would be nice to get rid of this dependency.